src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -1262,12 +1227,14 @@ void toString(TreePath p, StringBuilder sb) {
     void warnIfEmpty(DocTree tree, List<? extends DocTree> list) {
         for (DocTree d: list) {
             switch (d.getKind()) {
-                case TEXT:
+                case TEXT -> {
```

Comment:
Using `switch` here seems overkill.

Reply:
I almost fixed that one earlier. OK, I'll do that too. It was over-zealous use of use of unqualified names in an enum-switch